app/models/user.rb

```diff
@@ -57,6 +55,10 @@ def list_groups
     groups.map {|group| group.name }.join(', ')
   end
 
+  def cohort
+    Cohort.find(self.cohort_id)
+  end
```

Comment:
@srt32 Is the change from using a belongs_to to this cohort method related to GraphQL? I'm interested to learn why this is preferred.

Reply:
Also a good question. Not directly related to using graphql but more that the `cohort` record no longer exists in Census' database so we have to go across the network (which happens to be via graphql but could be json, could be xml, etc) to get it.

As a result we lose some of the niceties of ActiveRecord but gain the benefit of not duplicating data everywhere and (once we're done) we allow all the apps to get cohorts from a central place instead of having to store a copy (and keep that copy up to date) themselves.